fpga/bootrom/xip/Makefile

```diff
@@ -1,12 +1,17 @@
 # RISCV environment variable must be set
+# needs the following variables
+# LONG_NAME
+# BUILD_DIR
+# ROCKETCHIP_DIR
+# ROMCONF
 
 CC=$(RISCV)/bin/riscv64-unknown-elf-gcc
 OBJCOPY=$(RISCV)/bin/riscv64-unknown-elf-objcopy
 CFLAGS=-march=rv32imac -mabi=ilp32 -O2 -std=gnu11 -Wall -I. -nostartfiles -fno-common -g
 LFLAGS=-static -nostdlib
 
-dtb := $(BUILD_DIR)/$(CONFIG_PROJECT).$(CONFIG).dtb
-$(dtb): $(BUILD_DIR)/$(CONFIG_PROJECT).$(CONFIG).dts
+dtb := $(BUILD_DIR)/$(LONG_NAME).dtb
+$(dtb): $(BUILD_DIR)/$(LONG_NAME).dts
 	dtc -I dts -O dtb -o $@ $<
 
 .PHONY: dtb
```

Comment:
Hopefully we can apply the same `s/filename[^:]/filename_file/` rewrite rule to all these intermediate files.